.github/workflows/SBOM.yaml

```
@@ -0,0 +1,35 @@
+name: Generate SBOM
+
+on:
+  push:
+    
```

Tu va pas lancer ça a chaque push sur toutes les branches

```suggestion
on:
  release:
    types: [published]
```